packages/xo-server-test/src/_xoConnection.js

```diff
@@ -155,6 +162,20 @@ class XoConnection extends Xo {
     })
   }
 
+  async cloneTempVm(id) {
+    const clonedVmId = await this.call('vm.clone', {
```

Comment:
https://github.com/vatesfr/xen-orchestra/blob/26a295c8ed5fce313b735ec58d0c314ad29be3da/packages/xen-api/src/index.js#L244

Keep in mind that the ref-result must be extracted in this case: https://github.com/vatesfr/xen-orchestra/blob/26a295c8ed5fce313b735ec58d0c314ad29be3da/packages/xo-server/src/xapi/index.js#L422

Reply:
My bad, this is a call to XO.